### PR TITLE
Add radio field (sync with RDM)

### PR DESCRIPTION
**Pull request by fenekku**
Part of https://github.com/inveniosoftware/invenio-app-rdm/issues/122

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```
Comment (zzacharo) [inline .editorconfig:1]: @fenekku I am not sure we commit these files :) In react-searchkit we use `prettier` to format the js/html code. Just for consistency, I think I will setup this on this repo too...

File: README.md [hunk 2]
```diff
@@ -1 +1,82 @@
-# react-invenio-forms
\ No newline at end of file
+# react-invenio-forms
```
Comment (zzacharo) [inline README.md:1]: We said actually to remove the `create-react-app` related documentation from our repos so I will adjust the README in the react-invenio-deposit!

Comment (fenekku) [reply README.md:1]: Ah didn't know that :heavy_check_mark: . The main thing I added was how to run the tests.

File: .editorconfig (hunk 1 above)
Comment (fenekku) [reply .editorconfig:1]: Can editorconfig and prettier live side-by-side or has it been causing issues? Prettier is only for js. `.editorconfig` could be used for the rest (and basic js perhaps).

**Pull request merged**